src/server/stream_family.cc

```diff
@@ -1116,6 +1128,151 @@ OpResult<uint32_t> OpDel(const OpArgs& op_args, string_view key, absl::Span<stre
   return deleted;
 }
 
+struct PendingOpts {
+  string_view group_name;
+  string_view consumer_name;
+  ParsedStreamId start;
+  ParsedStreamId end;
+  int64_t min_idle_time = 0;
+  int64_t count = -1;
+};
+
+struct PendingReducedResult {
+  uint64_t count = 0;
+  streamID start;
+  streamID end;
+  vector<pair<string_view, uint64_t>> consumer_list;
```

Comment:
please add a comment what the number is `pair<string_view, uint64_t /* pending? */>`

Reply:
Ah, sure! It is the size of consumer pending list.